vmm/src/memory_manager.rs

```diff
@@ -249,7 +249,8 @@ impl MemoryManager {
             if hotplug_method == HotplugMethod::VirtioMem {
                 let start_addr = GuestAddress(
                     (start_of_device_area.0 + vm_virtio::VIRTIO_MEM_DEFAULT_BLOCK_SIZE - 1)
-                        & (!(vm_virtio::VIRTIO_MEM_DEFAULT_BLOCK_SIZE - 1)),
+                        / vm_virtio::VIRTIO_MEM_DEFAULT_BLOCK_SIZE
```

Comment:
Could you change this part to use bit operation?

Reply:
Your original code is right.  After I digged into the virtio-mem kernel code, I found that the block size actually doesn't have to be the power of 2 and I did not find any limitation on the block size. :)

Reply:
@uMartinXu so what does this mean? Are you gonna close this PR?

Reply:
This PR is still needed. 
My point is this PR just restore @teawater's original implementation.

Reply:
> This PR is still needed.
> My point is this PR just restore @teawater's original implementation.

My point is this code is OK. 
But It would be better if it could use bit operation.

Reply:
@teawater agree with you.
but as I mentioned above, it is easy and readable to use bit operation when block size is the power of 2. 
But I can't find a readable bit operation to handle this situation when block size can be any number.